### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by kritic3**

### CODE REVIEW

**Review by jlevine18 — CHANGES_REQUESTED** [review]
This looks very good so far- I would recommend also adding a section on finding good choices for the h parameter (sqrt of machine eps is often recommended, and some info is in Mariana's slides) as well as a couple graphical examples. Also, you might want to add a little more information on error bounds so that students have a bit more intuition on what's creating them. TYSM for everything so far.

File: notes/finite-difference.md [hunk 1]
```diff
@@ -13,55 +27,260 @@ sort: 14
 
 ## Finite Difference Approximation
 
-For a differentiable function \\(f:\mathbb{R} \rightarrow \mathbb{R}\\), the derivative is defined as
+#### Motivation
+
+For a given smooth function \\(f(x)\\), we want to calculate the derivative \\(f'(x)\\) at a given value of \\(x\\).
```
[the remark this answers is not in the document]
Comment (kritic3) [reply notes/finite-difference.md:32]: I think knowing the definition of a smooth function is part of the prerequisite knowledge for this class

File: notes/finite-difference.md [hunk 2]
```diff
@@ -13,55 +27,260 @@ sort: 14
...
+
+However, sometimes we do not know how to compute the analytical expression \\(f'(x)\\) or it is computationally too expensive. 
+
+Finite difference methods can help find an approximation for the value of \\(f'(x)\\).
+
+#### Definition
+
+For a differentiable function \\(f(x):\mathbb{R} \rightarrow \mathbb{R}\\), the derivative is defined as
```
[the remark this answers is not in the document]
Comment (kritic3) [reply notes/finite-difference.md:40]: I define that at the end. I thought defining a single variable was easier to understand before bringing up multivariable approximation. Should I mention it above too?

**Review by jlevine18 — APPROVED** [review]
Looks great; thank you so much for doing this

**Pull request merged**